src/engagement_db_to_analysis/code_imputation_functions.py

```diff
@@ -1,15 +1,66 @@
 from core_data_modules.util import TimeUtils
-
 from core_data_modules.data_models.code_scheme import CodeTypes
 from core_data_modules.cleaners.cleaning_utils import CleaningUtils
 from core_data_modules.traced_data import Metadata
 from core_data_modules.logging import Logger
-from src.engagement_db_to_analysis.column_view_conversion import get_latest_labels_with_code_scheme
+from core_data_modules.cleaners import Codes
+
 from engagement_database.data_models import Message
 
+from src.engagement_db_to_analysis.column_view_conversion import (get_latest_labels_with_code_scheme,
+                                                                  analysis_dataset_config_for_message)
+
 
 log = Logger(__name__)
 
+def _impute_not_reviewed_labels(user, messages_traced_data, analysis_dataset_configs):
+    """
+    Imputes Codes.NOT_REVIEWED label for messages that have not been manually checked in coda.
+
+    :param user: Identifier of user running the pipeline.
+    :type user: str
+    :param messages_traced_data: Messages TracedData objects to impute age_category.
+    :type messages_traced_data: list of TracedData
+    :param analysis_dataset_configs: Analysis dataset configuration in pipeline configuration module.
```

Comment:
We'll need to update this for all the functions requiring this; `configurations`

Reply:
yes in a separate pr.